providers/openlineage/src/airflow/providers/openlineage/utils/utils.py

```diff
@@ -54,7 +54,12 @@
 from airflow.sensors.base import BaseSensorOperator
 from airflow.serialization.serialized_objects import SerializedBaseOperator
 from airflow.utils.module_loading import import_string
-from airflow.utils.session import NEW_SESSION, provide_session
+
+if not AIRFLOW_V_3_0_PLUS:
+    from airflow.utils.session import NEW_SESSION, provide_session
+else:
+    NEW_SESSION = None  # type: ignore[assignment]
+    provide_session = None  # type: ignore[assignment]
```

Comment:
Dont think this will be needed

Reply:
YEash should not be is we add iff below.